src/git/config.go

```diff
@@ -160,6 +160,15 @@ func (c *Configuration) GetCodeHostingOriginHostname() string {
 	return c.getLocalConfigValue("git-town.code-hosting-origin-hostname")
 }
 
+// GetDeleteRemoteBranches indicates whether to delete the remote branch after shipping.
+func (c *Configuration) GetDeleteRemoteBranches() bool {
```

Comment:
Thoughts on renaming this to `ShouldShipDeleteRemoteBranch`? (similar to `ShouldNewBranchPush` / `ShouldSyncUpstream`?

Reply:
Sorry I forgot to see this comment until now. Great catch to match the naming to the existing steps. Changed.